src/compiler/reviews.md

```diff
@@ -1,4 +1,156 @@
+>   * compile times
+>   * memory usage
+>
+> #### Checklist for reviewers
+> * Am I the **right person to review** this PR:
+>   * Do I understand the code well enough?
+>   * Would I be able to spot non-obvious side effects?
+>   * Would I be able to fix a bug introduced by this PR?
+>   * Can I do the review in a timely fashion?
+>   * Do I feel pressure to quickly approve the PR for some reason?
+>   * Am I impartial enough?
+> * Before merging:
+>   * Is the **PR message still accurate**?
+>   * Is the **commit history clean** enough?
+
+
+
+## Guidance for dealing with common situations
+
+In most cases common sense is enough for deciding how to apply this policy.
+However, sometimes there are gray areas where it is not immediately clear how to proceed.
+This section lists a few common cases together with guidance on how to deal with them.
+
+* ### I don't think I am a good fit for reviewing - what now?
+  It is completely normal that you get assigned a PR (via highfive or otherwise) but don't feel comfortable reviewing it.
+  Here is what you can do, depending on the concrete case:
+
+  - If the change seems really big or contentious, consider asking for an MCP (see below).
+  - If you know just the right person for the review, assign them via `r? @<github-name>`.
+    It's polite to leave a comment asking them if they can take over --
+    but you don't have to make sure beforehand that they can actually do it.
+  - If you don't know the code well or already have too much on your plate,
+    ask highfive to roll the dice again via `r? @rust-lang/compiler-contributors`.
+
+  You can also always ask specific compiler team members for help finding a reviewer.
+
+* ### It is unclear if something constitutes a major change
+  Deciding if something is a "major change" requiring an [MCP][mcp] is not always straightforward.
+  The official guidelines are [here][whats-a-major-change].
+  When in doubt, err on side of treating something as a major change.
+  You can also nominate the PR for discussion in the compiler team's triage meeting by tagging it `I-nominated`.
+  If you nominate a PR please make sure to state a concrete question for the compiler team to discuss.
+
+* ### Intransparent discussion and rationale
+  Sometimes there are PRs that seem to be the result of some prior discussion, with no description or rationale.
+  They usually have a title like "Change X" and the only content of the PR message is "r? @xyz".
+  Even though the change might make sense and may even have been suggested by a compiler team member this is not good form.
+  The PR message should give a self-contained description of what is being changed,
+  why it is being changed and anything else that might be of interest.
+  Try to put yourself in the shoes of someone who, a few years down the road,
+  needs to fix a bug related to the code touched by the PR and needs to reconstruct the rationale for the way things are.
```

Comment:
@pnkfelix & @wesleywiser,  you both signaled that you wanted to discuss the wording of this paragraph.

Reply:
I think the heading of the paragraph should be a description of the specific problem one encounters. At least that's what I've been going for for the other sections. Does that make sense? I'm happy to rephrase the heading if that makes it clearer though.

Reply:
@wesleywiser, do you still think we should change the title to  "Transparent discussion and rationale".

Reply:
I've changed the title "Discussion or rationale is too intransparent", should be clearer.